sympy/matrices/tests/test_sparse.py

```diff
@@ -316,6 +316,7 @@ def sparse_zeros(n):
         SparseMatrix([(0, 1, 2), (3, 1, 2), (3, 4, 2), (3, 4, 5)])
     assert m1.reshape(2, 6) == \
         SparseMatrix([(0, 1, 2, 3, 1, 2), (3, 4, 2, 3, 4, 5)])
+    assert sparse_eye(10**3).reshape(1, 10**6).shape == (1, 10**6)  # is not slow
```

Comment:
I think it would be better just to remove this test. It isn't slow enough on master that it would get noticed in a PR. We can use benchmarks if we want to catch performance regressions.